lib/galaxy/model/store/discover.py

```diff
@@ -166,7 +166,7 @@ def create_dataset(
         if tag_list:
             job = self.get_job()
             self.tag_handler.add_tags_from_list(
-                job and job.user, primary_data, tag_list, flush=False, galaxy_session=job and job.galaxy_session
+                job and job.user, primary_data, tag_list, flush=False,
```

Comment:
We're setting up the tag handler in the various `ModelPersistenceContext` instances, so it wasn't needed to pass this explicitly.

Reply:
Thanks,  I was certainly confused by the extra passing parameter in those cases :sweat_smile: